src/main/java/core/basesyntax/strategy/operation/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void performOperation(FruitTransaction transaction) {
+        Storage.storage.put(transaction.getFruitName(), transaction.getQuantity());
+    }

```

Comment:
According to the checklist, the `BalanceOperation` class should not start with an empty line. Please remove the empty line at the beginning of the class implementation.